website/docs/guides/fusion-package-compat.md

```diff
@@ -0,0 +1,169 @@
+
+3. Run `dbt-autofix` in your package directory so it automatically updates your package code and rewrites YAML to conform to the latest JSON schema:
+    ```bash
+    dbt-autofix deprecations
+    ```
+
+4. (Optional) You can also run `dbt-autofix` as a temporary continuous integration (CI) check until the package is <Constant name="fusion"/>-compatible:
+    ```bash
+    dbt-autofix deprecations --check
+    ```
+
+### Test package with Fusion
+
+Now that you've run `dbt-autofix`, let's test your package with <Constant name="fusion"/> to ensure it's compatible before [updating](https://docs.getdbt.com//guides/fusion-package-compat?step=5) your `require-dbt-version` config. Refer to the [<Constant name="fusion"/> limitations documentation](/docs/fusion/supported-features#limitations) for more information on what to look out for. You can test your package by:
+
+<!-- no toc -->
+- [Running your integration tests with Fusion](#running-your-integration-tests-with-fusion)
+- [Manually validating your package](#manually-validating-your-package)
+
+#### Running your integration tests with Fusion
+
+If your package includes an `integration_tests/` folder ([like `dbt-utils`](https://github.com/dbt-labs/dbt-utils/tree/main/integration_tests)):
+1. Navigate to the folder (`cd integration_tests`) to run your tests. If you don't have an `integration_tests/` folder, you can either [create one](https://docs.getdbt.com/guides/building-packages?step=4) or navigate to the folder that contains your tests.
+2. Then, run your tests with <Constant name="fusion"/> by running the following `dbtf build` command (or whatever <Constant name="fusion"/> executable is available in your environment).
+3. If there are no errors, your package likely supports <Constant name="fusion"/> and you're ready to [update your `require-dbt-version`](https://docs.getdbt.com//guides/fusion-package-compat?step=5#update-your-require-dbt-version). If there are errors, you'll need to fix them first before updating your `require-dbt-version`.
+
+#### Manually validating your package
+
+If you don’t have integration tests:
+
+1. Create a small, <Constant name="fusion"/>-compatible dbt project that installs your package. 
+2. Run it with <Constant name="fusion"/> using the `dbtf run` command.
+3. Confirm that models build successfully and that there are no warnings. If there are errors/warnings, you'll need to fix them first. If you still have issues, reach out to the [#package-ecosystem channel](https://getdbt.slack.com/archives/CU4MRJ7QB) on Slack for help.
+
+### Update `require-dbt-version` 
+
+You should only update the [`require-dbt-version` config](/reference/project-configs/require-dbt-version) after testing and confirming that your package works with <Constant name="fusion"/>. 
+
+1. Update the `require-dbt-version` in your `dbt_project.yml` to include `2.0.0`. We recommend using a range to ensure stability across releases:
```

Comment:
do we want to mention they should do they in their packages.yml or should autofix do this?